Could not find `class-count-by-prefix.sparql`

I generated a repository the first time using ODK v1.4, and now I had to run a release and got an issue related to the SPARQL file mentioned in the title. 

```
robot --catalog catalog-v001.xml query -f tsv --use-graphs true -i tmp/afpo-preprocess.owl -s ../sparql/basic-report.sparql reports/basic-report.tsv -s ../sparql/class-count-by-prefix.sparql reports/class-count-by-prefix.tsv -s ../sparql/edges.sparql reports/edges.tsv -s ../sparql/xrefs.sparql reports/xrefs.tsv -s ../sparql/obsoletes.sparql reports/obsoletes.tsv -s ../sparql/synonyms.sparql reports/synonyms.tsv
File '../sparql/class-count-by-prefix.sparql' does not exist
Use the -vvv option to show the stack trace.
Use the --help option to see usage information.
make[1]: *** [Makefile:492: custom_reports] Error 1
make[1]: Leaving directory '/work/src/ontology'
make: *** [Makefile:312: prepare_release_fast] Error 2
```
I checked the commit history and found first the change on [Makefile](https://github.com/INCATools/ontology-development-kit/pull/558/files), and then [removing the sparql query without changing the Makefile](https://github.com/INCATools/ontology-development-kit/commit/fd9b5000dd9f0bc0d4118652109ead8278c44821). I don't see another sparql query with the same purpose.

Should the file be returned to the repository, or should its filename be removed from the Makefile?

--- a/odk/odk.py
+++ b/odk/odk.py
@@ -397,7 +397,7 @@ class ReportConfig(JsonSchemaMixin):
         The custom sparql checks available are: 'owldef-self-reference', 'redundant-subClassOf', 'taxon-range', 'iri-range', 'iri-range-advanced', 'label-with-iri', 'multiple-replaced_by', 'term-tracker-uri', 'illegal-date', 'dc-properties'.
     """
 
-    custom_sparql_exports : Optional[List[str]] = field(default_factory=lambda: ['basic-report', 'edges', 'xrefs', 'obsoletes', 'synonyms'])
+    custom_sparql_exports : Optional[List[str]] = field(default_factory=lambda: ['basic-report', 'class-count-by-prefix', 'edges', 'xrefs', 'obsoletes', 'synonyms'])
     """Chose which custom reports to generate. The related sparql query must be named CHECKNAME.sparql, and be placed in the src/sparql directory."""
 
     sparql_test_on: List[str] = field(default_factory=lambda: ['edit'])

--- a/template/src/sparql/class-count-by-prefix.sparql
+++ b/template/src/sparql/class-count-by-prefix.sparql
@@ -1,0 +1,10 @@
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix obo: <http://purl.obolibrary.org/obo/>
+
+SELECT ?prefix (COUNT(DISTINCT ?cls) AS ?numberOfClasses) WHERE 
+{
+  ?cls a owl:Class .
+  FILTER (!isBlank(?cls))
+  BIND( STRBEFORE(STRAFTER(str(?cls),"http://purl.obolibrary.org/obo/"), "_") AS ?prefix)
+}
+GROUP BY ?prefix

